clang/test/CodeGenCXX/microsoft-abi-eh-async.cpp

```diff
@@ -0,0 +1,182 @@
+// REQUIRES: x86-registered-target
+
+// RUN: %clang_cl -c --target=x86_64-windows-msvc /EHa -O2 /GS- \
+// RUN:   -Xclang=-import-call-optimization \
+// RUN:   /clang:-S /clang:-o- %s 2>&1 \
```

Comment:
Does this need the `--` before the file name to avoid weird command line parsing issues?

Reply:
Added `--`